.github/workflows/static_checks.yml

```diff
@@ -106,5 +106,5 @@ jobs:
         uses: codespell-project/actions-codespell@v1
         with:
           skip: "./bin,./thirdparty,*.desktop,*.gen.*,*.po,*.pot,*.rc,./AUTHORS.md,./COPYRIGHT.txt,./DONORS.md,./core/input/gamecontrollerdb.txt,./core/string/locales.h,./editor/project_converter_3_to_4.cpp,./misc/scripts/codespell.sh,./platform/android/java/lib/src/com,./platform/web/node_modules,./platform/web/package-lock.json"
-          ignore_words_list: "curvelinear,doubleclick,expct,findn,gird,hel,inout,lod,nd,numer,ot,te,vai"
+          ignore_words_list: "curvelinear,doubleclick,expct,findn,gird,hel,inout,lod,nd,numer,ot,te,vai,requestor"
```

Comment:
Seems unrelated to this PR?
This list should also be kept in sync with `misc/scripts/codespell.sh`.

Reply:
Unrelated, but it's in the same file that was edited. Updated the script.

Reply:
Done